models/trainer/DCGAN_trainer.py

```diff
@@ -18,6 +19,7 @@ def getDefaultConfig(self):
 
     def __init__(self,
                  pathdb,
+                 miniBatchScheduler=None,                 
```

Comment:
Adding this argument is necessary because dataset.py sets up miniBatchScheduler and we get a crash at runtime because __init__ does not have such an argument ?

Reply:
It shouldn't crash unless you add a scheduler in the configuration file.

Reply:
Presumably there is a problem in the automatic generation of the configuration file.
@Molugan how should we proceed regarding this issue ? I would go for keeping my fix so that at least it does not crash.

Reply:
The issue is DCGAN + celebaHQ in the dataset maker. I hadn'y considered the possibility that someone would want to use both.

Reply:
Then the dataset maker should be changed, not this part.